Front-end/src/Profile.jsx

```diff
@@ -98,19 +100,83 @@ const Profile = (props) => {
       })
     );
   }
+  function Followers() {
+    fetch(`http://localhost:3000/followers/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        setFollowers(data);
+      })
+    );
+  }
+  function SetFollow() {
+
+    fetch(`http://localhost:3000/followUser/${user.id}/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        if (data === 0) {
+          setStatus("Follow");
+        } else {
+          setStatus("Unfollow");
+        }
+      })
+    );
+  }
+  function handleFollow(){
+    if (status === "Follow") {
+      console.log("TEST", profile.id)
```

Comment:
Could you check all console.log/commented code ? Feel free to have a separate commit to do that :)

Reply:
They were removed in an earlier commit.